plugins/inputs/ssl_cert/ssl_cert_test.go

```diff
@@ -0,0 +1,217 @@
+package ssl_cert
+
+import (
+	"crypto/tls"
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+const testCert = `-----BEGIN CERTIFICATE-----
```

Comment:
Can you update to use the testing certs in `testutil/pki`, check `plugins/inputs/socket_listener/socket_listener_test.go` for examples.